### PR TITLE
[Big Tensor] Remove configurations that may cause OOM for gaussian_nll_loss

### PR DESCRIPTION
**Pull request by HeyDavid633**
删除了 report 与 tester 中 会导致 [paddle.nn.functional.gaussian_nll_loss](https://www.paddlepaddle.org.cn/documentation/docs/zh/2.6/api/paddle/nn/functional/gaussian_nll_loss_cn.html#gaussian-nll-loss) OOM 的 case

### CODE REVIEW

**Review by wanghuancoder — APPROVED** [review]
LGTM
